validphys2/src/validphys/config.py

```diff
@@ -480,6 +480,31 @@ def parse_dataset_input(self, dataset: Mapping, allow_legacy_names: bool = True)
             sys=sysnum,
         )
 
+    def parse_inconsistent_data_settings(self, settings):
+        """
+        Parse the inconsistent data settings from the yaml file.
+        """
```

Comment:
can you add a description of inconsistent_data_settings and the meaning of the known keys?